src/main/java/org/folio/rest/impl/PermsAPI.java

```diff
@@ -1080,25 +1085,33 @@ public void deletePermsPermissionsById(String id,
   @SuppressWarnings("java:S3776")
   @Validate
   @Override
-  public void getPermsPermissions(String expandSubs, String expanded, String includeDummy,
+  public void getPermsPermissions(String expandSubs, String expanded, String includeDummy, String includeInactive,
                                   int length, int start, String sortBy, String query, String memberOf,
                                   String ownedBy, Map<String, String> okapiHeaders,
                                   Handler<AsyncResult<Response>> asyncResultHandler,
                                   Context vertxContext) {
 
     try {
       boolean includeDummyPerms = "true".equals(includeDummy);
-      String[] queryArr = new String[]{""};
+      boolean includeInactivePerms = "true".equals(includeInactive);
+      String[] queryArr = new String[]{query};
```

Comment:
`queryArr` is not (or no longer) used in a callback.
Simply rename the query parameter to query0.
Then use
`String query = query0 == null ? "" : query0;`